From 1c98142d552c5e260e61fe6b642f65a8ceb06d3b Mon Sep 17 00:00:00 2001
Message-Id: <1c98142d552c5e260e61fe6b642f65a8ceb06d3b.1484342411.git.ymankad@redhat.com>
In-Reply-To: <df26b13347392089e5699a6be872de7806a36a8c.1484342411.git.ymankad@redhat.com>
References: <df26b13347392089e5699a6be872de7806a36a8c.1484342411.git.ymankad@redhat.com>
From: Yash Mankad <ymankad@redhat.com>
Date: Fri, 13 Jan 2017 15:55:40 -0500
Subject: [CHANGE 5/8] Revert "iotests: Fix 026's reference output"
To: ymankad@redhat.com

This reverts commit 9ade1429890217e9e1460893250166e73b5f7ab6.

This is to revert Max's patch series that fix BZ#1405882 as
they did not have the 6.9 release flag set and could not be
justified as exceptions / blockers. Will be moved to 6.10.

Signed-off-by: Yash Mankad <ymankad@redhat.com>
---
 tests/qemu-iotests/026.out | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/tests/qemu-iotests/026.out b/tests/qemu-iotests/026.out
index 019aed9..f7c78e7 100644
--- a/tests/qemu-iotests/026.out
+++ b/tests/qemu-iotests/026.out
@@ -14,7 +14,6 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_update; errno: 5; imm: off; once: off; write 
-qcow2_free_clusters failed: Input/output error
 write failed: Input/output error
 
 1 leaked clusters were found on the image.
@@ -22,7 +21,6 @@ This means waste of disk space, but no harm to data.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_update; errno: 5; imm: off; once: off; write -b
-qcow2_free_clusters failed: Input/output error
 write failed: Input/output error
 
 1 leaked clusters were found on the image.
@@ -40,7 +38,6 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_update; errno: 28; imm: off; once: off; write 
-qcow2_free_clusters failed: No space left on device
 write failed: No space left on device
 
 1 leaked clusters were found on the image.
@@ -48,7 +45,6 @@ This means waste of disk space, but no harm to data.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_update; errno: 28; imm: off; once: off; write -b
-qcow2_free_clusters failed: No space left on device
 write failed: No space left on device
 
 1 leaked clusters were found on the image.
@@ -186,13 +182,11 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l2_alloc.write; errno: 5; imm: off; once: off; write 
-qcow2_free_clusters failed: Input/output error
 write failed: Input/output error
 No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l2_alloc.write; errno: 5; imm: off; once: off; write -b
-qcow2_free_clusters failed: Input/output error
 write failed: Input/output error
 
 1 leaked clusters were found on the image.
@@ -210,13 +204,11 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l2_alloc.write; errno: 28; imm: off; once: off; write 
-qcow2_free_clusters failed: No space left on device
 write failed: No space left on device
 No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l2_alloc.write; errno: 28; imm: off; once: off; write -b
-qcow2_free_clusters failed: No space left on device
 write failed: No space left on device
 
 1 leaked clusters were found on the image.
@@ -571,7 +563,6 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_grow.write_table; errno: 5; imm: off; once: off
-qcow2_free_clusters failed: Input/output error
 write failed: Input/output error
 No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
@@ -582,7 +573,6 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_grow.write_table; errno: 28; imm: off; once: off
-qcow2_free_clusters failed: No space left on device
 write failed: No space left on device
 No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
@@ -593,7 +583,6 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_grow.activate_table; errno: 5; imm: off; once: off
-qcow2_free_clusters failed: Input/output error
 write failed: Input/output error
 
 96 leaked clusters were found on the image.
@@ -606,7 +595,6 @@ No errors were found on the image.
 Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=1073741824 
 
 Event: l1_grow.activate_table; errno: 28; imm: off; once: off
-qcow2_free_clusters failed: No space left on device
 write failed: No space left on device
 
 96 leaked clusters were found on the image.
-- 
2.7.4

